config.py

```diff
@@ -5,6 +5,10 @@
 """
 import os
 
+from dotenv import load_dotenv
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+load_dotenv(os.path.join(basedir, '.flaskenv'))
```

Comment:
I think this is actually automatic, dotenv looks for this file in the base directory.  But that being said, never hurts to be specific.